src/config.cpp

```diff
@@ -134,6 +134,47 @@ namespace {
 // to be replaced by std::filesystem once C++17 is used
 namespace fs = ghc::filesystem;
 
+void checkBiophysicalPopulations(
+    const std::unordered_map<std::string, PopulationProperties>& populations) {
+    for (const auto& it : populations) {
+        const auto population = it.first;
```

Comment:
Will this take a copy; and if yes, is that intentional?

Reply:
tnx, I'll fix that